script.js

```diff
@@ -26,3 +27,136 @@ contact.addEventListener('submit', (event) => {
+    technologies: ['html', 'css', 'javscript', 'github', 'ruby', 'Bootstraps'],
+    company: 'Uber',
+    role: 'Lead developer',
+    year: '2018',
+    linkLive: 'https://naolkidanu.github.io/portfolio/',
+    linkSource: 'https://github.com/naolkidanu/portfolio',
+  },
+];
+for (let i = 0; i < mainData.length; i += 1) {
+  const card = `<div class="card">
+      <img class="pic" src="${mainData[i].imageUrl}" alt="First-img" />
+      <div class="Desktop-common">
+      <h2 class="card-name">${mainData[i].name}</h2>
+      <ul class="can-bak">
+        <li class="cann">${mainData[i].company}</li>
+        <li class="bak">${mainData[i].role}</li>
+        <li class="bak">${mainData[i].year}</li>
+      </ul>
+      <p class="tex-sec">${mainData[i].description}</p>
+      <ul class="work-list">
+      <li class="work-name">${mainData[i].technologies[0]}</li>
+        <li class="work-name">${mainData[i].technologies[1]}</li>
+        <li class="work-name">${mainData[i].technologies[2]}</li>
+      </ul>
+      <button class="sec-button">see project</button>
+    </div>
+    </div>`;
+  project.insertAdjacentHTML('beforeend', card);
+}
+
+const modalButton = document.querySelectorAll('.sec-button');
+for (let i = 0; i < modalButton.length; i += 1) {
+  modalButton[i].addEventListener('click', () => {
+    modalEl.classList.toggle('hidden');
+    wrapper.classList.toggle('blur');
+    overFlow.classList.toggle('scroll-off');
+
+    let loopInner;
+    mainData[i].technologies.forEach((tech) => {
+      loopInner += `<li class="work-name">${tech}</li>`;
+      return loopInner;
+    });
+
+    modalEl.innerHTML = `<div class="card1">
+      <div class="Desktop-common1">
+        <div class="head-just1"><h2 class="card-name1">${mainData[i].name}</h2><img src="img/Icon-Cancel.svg" alt="Cancel" class="cancel icon-img"/></div><ul class="can-bak1">
+          <li class="cann1">${mainData[i].company}</li><li class="bak1">${mainData[i].role}</li><li class="bak1">${mainData[i].year} </li>
+        </ul>
+        <img
+          class="pic1"
+          src=${mainData[i].imageUrl}
+          alt="First-img"
+        />
+        <div class="One1">
+          <p class="tex-sec1">${mainData[i].description}</p>
+          <div class="two1">
+            <ul class="work-list1">
+               <li class="work-name">${mainData[i].technologies[0]}</li>
+               <li class="work-name">${mainData[i].technologies[1]}</li>
+               <li class="work-name">${mainData[i].technologies[2]}</li>
+            </ul>
+            <ul class="live1">
+              <a href=${mainData[i].linkLive} class="sec-button1">
+                see live
+                <img src="img/Vector-git.png" class="vector1" />
+              </a>
+              <a href=${mainData[i].linkSource} class="sec-button1">
+                see source<img src="img/Icon-s.png" class="vector1" />
+              </a>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>`;
```

Comment:
### Your Desktop Popup
![Student Popup](https://user-images.githubusercontent.com/93039359/197288326-1d7d774c-efa0-493c-8b2f-c21abb8a610d.PNG)
### Figma Desktop Popup
![Figma Popup](https://user-images.githubusercontent.com/93039359/197288371-8eb9d599-0728-4b39-ba8b-4d7d19cb2872.PNG)

- [ ] In your desktop popup you can see that first it is not properly positioned in the center as you can see that in the Figma template 🤔 Kindly make it positioned according to the Figma template 👍
- [ ] Second your contents are not properly styled in the Desktop version. Kindly make your whole desktop popup look similar to the Figma template 👍 
- [ ] You can compare both images and you will notice that your popup is supposed to be a card with white background and there is an grey overlay behind this card with blur effect. Kindly implement these things to make your popup window consistent with the Figma template 👍